experimental/SetPartitions/src/SetPartition.jl

```diff
@@ -404,3 +404,109 @@ function compose_count_loops(p::SetPartition, q::SetPartition)
     
     return (ret, length(related_comp))
 end
+
+"""
+    number_of_blocks(V::SetPartition)
+
+Return the number of blocks of a `SetPartition`.
```

Comment:
```suggestion
    number_of_blocks(p::SetPartition)

Return the number of blocks of `p`.
```
consistency with other docstrings in this file